Presentation/DualNavigationControllersSplitDelegate.swift

```diff
@@ -12,8 +12,25 @@ import Flow
 /// A split controller delegate (`UISplitViewControllerDelegate`), that manages navigation controllers for the master and detail view,
 /// as well as moving view controllers between these while expanding or collapsing.
 open class DualNavigationControllersSplitDelegate: NSObject, UISplitViewControllerDelegate {
-    private let isCollapsedProperty = ReadWriteSignal(true)
+
+    @available(*, deprecated, message: "use `init(collapsedState:)` instead")
+    public convenience override init() {
+        self.init(collapsedState: Future(true))
+    }
+
+    /// Creates a delegate that will manage a split view controller with the specified initial collapsed state
+    /// - Parameter collapsedState: Use to report the initial `isCollapsed` state of the split view controller.
+    /// Note that its value can be unreliable until it is rendered on the screen.
+    public init(collapsedState: Future<Bool>) {
```

Comment:
Any reason we pass this as a future instead of as a signal? Sound more like something that might change over time than something that takes time to complete? Rename to `isCollapsed`?

Reply:
The source of truth for the collapsed state is the split view controller. I think its `isCollapsed` property is not kvo compliant. It seems like once it's on screen the delegate methods track properly the collapsing/expanding so I modelled it as a future to make it clear it's only the first time it gets added to a window that we don't know what's the state.

Reply:
I agree for the renaming.